sotodlib/toast/workflows/sim_atm.py

```diff
@@ -78,6 +78,10 @@ def simulate_atmosphere_signal(job, otherargs, runargs, data):
         None
 
     """
+    log = toast.utils.Logger.get()
```

Comment:
Isn't this handled by the decorator?

Reply:
There is an additional logger use inside this function, outside of the timing decorator.